### ISSUE TITLE
Doorbell silent mode switch unavailable in v0.7.2b5

### ISSUE THREAD

**Issue by visualage** [twrecked/hass-aarlo#540]
Doorbell silent mode switch entities cannot be registered in v0.7.2b5. It is working fine in v0.7.2b4. It seem the problem is in https://github.com/twrecked/hass-aarlo/blob/4ac232131b58e9b33ee3cc265b92c03e9133f302/custom_components/aarlo/switch.py#L165-L173

and 

https://github.com/twrecked/hass-aarlo/blob/4ac232131b58e9b33ee3cc265b92c03e9133f302/custom_components/aarlo/switch.py#L373-L378

The `AarloSwitch.device_info` method expects `self._device` to be set. However, in `AarloSilentModeBaseSwitch.__init__`, `self._device` is never set. I suspect a simple `self._device = doorbell;` in `AarloSilentModeBaseSwitch.__init__` may fix it, but would like to confirm this is indeed a reasonable fix.

**Comment by twrecked** [comment]
The release was actually built from the wrong branch. Can you try now?

Having said that, I will look and see why this is broken and fix it in the 0.8 branch.

**Comment by visualage** [comment]
Yup. It works now with v0.7.2b5.1. If you would like to close this, you can.
Looks like v0.7.2b5 was released from master branch? The code above is in both master and v0.7.2b5 branches.
